Source/ThirdParty/ANGLE/src/libANGLE/State.h

```diff
@@ -625,6 +625,11 @@ class State : angle::NonCopyable
     void setPixelLocalStorageActivePlanes(GLsizei n);
     GLsizei getPixelLocalStorageActivePlanes() const { return mPixelLocalStorageActivePlanes; }
 
+    // GL_ANGLE_variable_rasterization_rate_metal
```

Comment:
Two variants:
GL_ANGLE_variable_rasterization_rate_metal
GL_ANGLE_metal_variable_rasterization_rate

Reply:
That's what I get for renaming the extension 5 times!

Reply:
@kkinnunen-apple @kenrussell

> GL_ANGLE_variable_rasterization_rate_metal

I've standardized on this variant, since there could be a general `GL_ANGLE_variable_rasterization_rate` or platform specific `GL_ANGLE_variable_rasterization_rate_xxx` in the future.

Unless you think that the differences between platforms is too great for there to ever be a commonality? (As witnessed by nVidia and QCOM shading rate extensions)

Reply:
`GL_ANGLE_variable_rasterization_rate_metal` sounds fine.